src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java

```diff
@@ -165,6 +174,11 @@ public void onBuildStart(AbstractBuild<?, ?> build,
         } else {
             sb.append(Ghprb.replaceMacros(build, listener, startedStatus));
         }
+
+        if(showMatrixStatus && !(build.getProject() instanceof MatrixProject)){
```

Comment:
If we were to create a Freestyle project with the checkbox unclicked, then ```if(showMatrixStatus && !(build.getProject() instanceof MatrixProject)){``` would return false due to the ```showMatrixStatus``` variable. , and it would not enter the if statement, and go straight to going to creating a commit status for the job. 

```
if(showMatrixStatus && !(build.getProject() instanceof MatrixProject)){
     return;
}
createCommitStatus(build, listener, sb.toString(), repo, GHCommitState.PENDING);
```